### ISSUE TITLE
Installation documentation 

### ISSUE THREAD

**Issue by i64** [Battery-Intelligence-Lab/DTWpp#17]
Hi authors, I am reviewing the code from https://github.com/openjournals/joss-reviews/issues/6881.

In installation
`cd DTWC++  is wrong it should be cd dtw-cpp`

running is itself the installation, and overview, installation, running should be in order. Also -j32 should not be assumed where you could use this command in in linux to determine number of processors. 

`cmake --build . -j$(( $(nproc) * 2 -1))`

**Comment by ElektrikAkar** [comment]
Hi,

Thank you very much for your comment! We have fully updated the installation documentation https://github.com/Battery-Intelligence-Lab/dtw-cpp/pull/20.

- We corrected `DTWC++` to `dtw-cpp` as you suggested.
- Regarding the build process, we decided to retain `-j4` for two reasons:  
  i) It is clearer and more readable than `-j$(( $(nproc) * 2 - 1))`.  
  ii) Different operating systems may require different commands to check the number of processors. For example, on Windows, we were able to display the number of processors using the `echo %NUMBER_OF_PROCESSORS%` command; however, it did not support mathematical operations like `2* %NUMBER_OF_PROCESSORS% - 1`. Therefore, we prefer to keep the command as `-j4`. However, we have added an explanation to clarify the reasoning behind the number `4` and included `-j$(( $(nproc) * 2 - 1))` in the [Linux section](https://battery-intelligence-lab.github.io/dtw-cpp/1_getting_started/1_installation.html#linux-debian--ubuntu-2004). We also reduced the number from `-j32` to `-j4` to better accommodate users with lower-spec systems.
